text/src/csplit.rs

```diff
@@ -59,10 +59,6 @@ struct SplitOps {
     ops: Vec<Operand>,
 }
 
-fn inc_char(ch: char) -> char {
-    ((ch as u8) + 1) as char
-}
-
 struct OutputState {
```

Comment:
Undocumented. What's `in_line_no`? Why are we starting from 1?

Reply:
"this is because lines in the file are numbered from 1 rather than zero, for historical reason"
It is written in the documentation for the utility